scipy/ndimage/_filters.py

```diff
@@ -984,13 +1041,15 @@ def uniform_filter(input, size=3, output=None, mode="reflect",
     input = numpy.asarray(input)
     output = _ni_support._get_output(output, input,
                                      complex_output=input.dtype.kind == 'c')
-    sizes = _ni_support._normalize_sequence(size, input.ndim)
-    origins = _ni_support._normalize_sequence(origin, input.ndim)
-    modes = _ni_support._normalize_sequence(mode, input.ndim)
-    axes = list(range(input.ndim))
+    axes = _ni_support._check_axes(axes, input.ndim)
+    num_axes = len(axes)
+    sizes = _ni_support._normalize_sequence(size, num_axes)
+    origins = _ni_support._normalize_sequence(origin, num_axes)
+    modes = _ni_support._normalize_sequence(mode, num_axes)
     axes = [(axes[ii], sizes[ii], origins[ii], modes[ii])
-            for ii in range(len(axes)) if sizes[ii] > 1]
+            for ii in range(num_axes) if sizes[ii] > 1]
     if len(axes) > 0:
+        print(f"{axes=}")
```

Comment:
```suggestion
```
I assume this was for debugging?

```
scipy.ndimage.uniform_filter
----------------------------

File "build/testenv/lib/python3.9/site-packages/scipy/ndimage/_filters.py", line 1085, in uniform_filter
Failed example:
    result = ndimage.uniform_filter(ascent, size=20)
Expected nothing
Got:
    axes=[(0, 20, 0, 'reflect'), (1, 20, 0, 'reflect')]
```

Reply:
thanks. planning to revert the non-Gaussian changes for now, so the suggestion will become outdated